src/index.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const express = require('express');
+
+require('dotenv').config();
+
+const { usersRouter } = require('./routes/users.route');
+const { expensesRouter } = require('./routes/expenses.route');
+
+const PORT = process.env.API_PORT;
```

Comment:
what if `process.env.API_PORT` is undefined?
```suggestion
const PORT = process.env.API_PORT ?? 3001;
```

Reply:
Unfortunately I can use logical OR `||` only due linter setup